tests/src/test/scala/services/HeadersTests.scala

```diff
@@ -67,15 +66,19 @@ class HeadersTests extends FlatSpec with Matchers with ScalaFutures with WskActo
   val allMethods = Some(Set(DELETE.name, GET.name, POST.name, PUT.name))
   val allowOrigin = `Access-Control-Allow-Origin`.*
   val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type")
-  val url = Uri(s"http://${WhiskProperties.getBaseControllerAddress()}")
+  val url = Uri(s"https://${WhiskProperties.getBaseControllerAddress()}")
 
   def request(method: HttpMethod, uri: Uri, headers: Option[Seq[HttpHeader]] = None): Future[HttpResponse] = {
     val httpRequest = headers match {
       case Some(headers) => HttpRequest(method, uri, headers)
       case None          => HttpRequest(method, uri)
     }
+    val sslConfig = AkkaSSLConfig().mapSettings { s =>
+      s.withHostnameVerifierClass(classOf[AcceptAllHostNameVerifier].asInstanceOf[Class[HostnameVerifier]])
+    }
```

Comment:
Can we do this in a generalized way since our Rest testcases need the same "functionality". Maybe we can have a `testReques(req: HttpRequest)` which does the same as 

```scala
Http().singleRequest(httpRequest, connectionContext = connectionContext)
```